pkg/buildscript/mutations.go

```diff
@@ -143,7 +159,7 @@ type PlatformNotFoundError struct {
 	*locale.LocalizedError // for legacy non-user-facing error usages
 }
 
-func (b *BuildScript) removePlatform(platformID strfmt.UUID) error {
+func (b *BuildScript) RemovePlatform(platformID strfmt.UUID) error {
```

Comment:
By changing the visibility of this method, it's possible that `UpdatePlatform()` is no longer needed. Please consider removing it.

Reply:
Oh this one I actually can remove! Thanks for catching.